onnxruntime/core/mlas/lib/mlasi.h

```diff
@@ -563,6 +576,91 @@ MlasMinimumFloat32x4(MLAS_FLOAT32X4 Vector1, MLAS_FLOAT32X4 Vector2)
 #endif
 }
 
+inline
+MLAS_FLOAT32X4
+MlasGreaterThanFloat32x4(MLAS_FLOAT32X4 Vector1, MLAS_FLOAT32X4 Vector2)
+{
+#if defined(MLAS_NEON_INTRINSICS)
+    return vreinterpretq_f32_u32(vcgtq_f32(Vector1, Vector2));
```

Comment:
Just to confirm, does this build with Windows ARM/ARM64 and Linux ARM/ARM64? And x86?

Reply:
Not testing NEON part yet. Will do extra round of work on it.

Reply:
Build pass all arm related target on windows and Linux.